### DISCUSSION TITLE
Batch file to push settings and file associations

### DISCUSSION THREAD

**Discussion by WLODDP** [d2phap/ImageGlass#1083] [Q&A]
I am wrapping a win32 app (.intunewin) to be deployed with Intune after I deploy ImageGlass MSI via Intune too.

The app is basically a batch file and the xmlconfig file, with two lines:

```
COPY  /Y .\igconfig.xml %APPDATA%\ImageGlass

C:\Program Files\ImageGlass\igtasks.exe regassociations "*.avif;*.b64;*.bmp;*.cur;*.cut;*.dds;*.dib;*.emf;*.exif;*.gif;*.heic;*.heif;*.ico;*.jfif;*.jp2;*.jpe;*.jpeg;*.jpg;*.jxl;*.pbm;*.pcx;*.pgm;*.png;*.ppm;*.psb;*.svg;*.tif;*.tiff;*.webp;*.wmf;*.wpg;*.xbm;*.xpm;*.exr;*.hdr;*.psd;*.tga;*.3fr;*.ari;*.arw;*.bay;*.crw;*.cr2;*.cr3;*.cap;*.dcs;*.dcr;*.dng;*.drf;*.eip;*.erf;*.fff;*.gpr;*.iiq;*.k25;*.kdc;*.mdc;*.mef;*.mos;*.mrw;*.nef;*.nrw;*.obm;*.orf;*.pef;*.ptx;*.pxn;*.r3d;*.raf;*.raw;*.rwl;*.rw2;*.rwz;*.sr2;*.srf;*.srw;*.x3f;"
```

For some reasons it doesn't seem to be working, has someone tried this before? 

Thanks!

**Comment by d2phap** [comment]
Maybe you need `"` for the path?
```
"C:\Program Files\ImageGlass\igtasks.exe" regassociations  "*.avif;*.b64;"
```

**Reply by WLODDP** [reply]
oh, I didn't think of that... I will give it a go!
Thanks!

**Reply by WLODDP** [reply]
Unfortunately didn't make any difference...

**Reply by fire-eggs** [reply]
If you run the command in a command window, what happens? The first time I tried to run it, it was in a context where it didn't have a valid igtheme.xml file, and crashed.

Also, igtasks requires admin privs.
